src/queries/disputes.js

```diff
@@ -0,0 +1,107 @@
+import gql from 'graphql-tag'
+
+export const AllDisputes = gql`
+  subscription {
+    disputes(orderBy: createdAt, orderDirection: desc) {
+      id
+      createTermId
+      possibleRulings
+      finalRuling
+      lastRoundId
+      state
+      metadata
+      createdAt
+      subject {
+        id
+      }
+      rounds {
+        state
+        number
+        draftTermId
+        jurorsNumber
+        settledPenalties
+        jurorFees
+        delayedTerms
+        selectedJurors
+        coherentJurors
+        collectedTokens
+        createdAt
+        jurors {
+          juror {
+            id
+          }
+        }
+        appeal {
+          id
+          maker
+          appealedRuling
+          taker
+          opposedRuling
+          settled
+          createdAt
+        }
+      }
+    }
+  }
```

Comment:
I'd only query the data we need to work with, it seems like we are querying all the information related for each dispute when we probably need only a few things for the lists disputes page